src/App.js

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Pokemon from './components/Pokemon';
+import Button from './components/Button';
 import './App.css';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemons, setPokemons] = useState([]);
```

Comment:
Als `pokemons` een object gaat worden, dan zou ik als default waarde geen array pakken, maar undefined. Dat past beter bij het type wat het gaat worden
```suggestion
    const [pokemons, setPokemons] = useState();
```